src/main/java/org/gradle/playframework/tasks/WebJarsExtract.java

```diff
@@ -0,0 +1,60 @@
+package org.gradle.playframework.tasks;
+
+import org.gradle.api.DefaultTask;
+import org.gradle.api.file.ConfigurableFileCollection;
+import org.gradle.api.file.Directory;
+import org.gradle.api.provider.Property;
+import org.gradle.api.tasks.InputFiles;
+import org.gradle.api.tasks.OutputDirectory;
+import org.gradle.api.tasks.TaskAction;
+import org.gradle.playframework.tasks.internal.WebJarsExtractRunnable;
+import org.gradle.playframework.tools.internal.webjars.DefaultWebJarsExtractSpec;
+import org.gradle.playframework.tools.internal.webjars.WebJarsExtractSpec;
+import org.gradle.workers.IsolationMode;
+import org.gradle.workers.WorkerExecutor;
+
+import javax.inject.Inject;
+
+public class WebJarsExtract extends DefaultTask {
+    private final WorkerExecutor workerExecutor;
+    private final Property<Directory> outputDirectory;
+    private final ConfigurableFileCollection webJarsClasspath;
+    private final ConfigurableFileCollection webJarsExtractorClasspath;
+
+    @Inject
+    public WebJarsExtract(WorkerExecutor workerExecutor) {
+        this.workerExecutor = workerExecutor;
+        this.outputDirectory = getProject().getObjects().directoryProperty();
+        this.webJarsClasspath = getProject().files();
+        this.webJarsExtractorClasspath = getProject().files();
+    }
+
+    @OutputDirectory
+    public Property<Directory> getOutputDirectory() {
+        return outputDirectory;
+    }
+
+    @InputFiles
+    public ConfigurableFileCollection getWebJarsClasspath() {
+        return webJarsClasspath;
+    }
+
+    public ConfigurableFileCollection getWebJarsExtractorClasspath() {
+        return webJarsExtractorClasspath;
+    }
+
+    @TaskAction
+    public void extract() {
+        final WebJarsExtractSpec spec = new DefaultWebJarsExtractSpec(getWebJarsClasspath().getFiles(), getOutputDirectory().get().getAsFile());
+
+        workerExecutor.submit(WebJarsExtractRunnable.class, workerConfiguration -> {
+            workerConfiguration.setIsolationMode(IsolationMode.PROCESS);
+            workerConfiguration.forkOptions(options -> options.jvmArgs("-XX:MaxMetaspaceSize=256m"));
```

Comment:
Shouldn't this be the default, but still configurable? It looks like it's just the default currently, but not configurable.

Reply:
Ah... I just followed existing codebase: https://github.com/gradle/playframework/blob/master/src/main/java/org/gradle/playframework/tasks/JavaScriptMinify.java#L84 🙈 

Didn't have bandwidth to make this configurable :(

Reply:
If this is already not configurable elsewhere, this is probably fine.